server/pull_request.go

```diff
@@ -217,6 +228,18 @@ func handlePRUnlabeled(pr *model.PullRequest, removedLabel string) {
 			}
 		}
 
+		// Old comments created by Mattermod user will be deleted here.
+		mlog.Info("Removing old Mattermod comments")
+		for _, comment := range comments {
```

Comment:
Maybe we should limit this to just removing comments related to the test server? Mattermod occasionally posts other useful things like messages asking people to sign the CLA

Reply:
@hmhealey Thanks a lot for the review. Replacing "if *comment.User.Login == Config.Username" with "if *comment.User.Login == Config.Username && (*comment.Body == Config.SetupSpinmintMessage || *comment.Body == Config.SetupSpinmintUpgradeMessage || *comment.Body == Config.SetupSpinmintDoneMessage || *comment.Body == Config.SetupSpinmintFailedMessage)" will remove only test server related comments. It really depends whether we want to keep this kind of messages as I thought that all messages after a label is removed are of no real need. @hmhealey @cpanato What do you think? Would you prefer to keep server non related messages?

Reply:
we can remove the CLA messages if mattermod detected the CLA was signed.

otherwise i agree we should delete only the test server messages.

Reply:
and looks ok

Reply:
@hmhealey @cpanato The script was updated to remove only test server related messages.